### DISCUSSION TITLE
自己训练的版面分析模型如何配合表格识别使用

### DISCUSSION THREAD

**Discussion by Xrtero** [PaddlePaddle/PaddleOCR#12693] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

如何将训练好的模型与其他模型进行结合，有相关文档吗，以及后续如何部署呢？

- 系统环境/System Environment：Linux
- 版本号/Version：Paddle：  2.4.2
- 运行指令/Command Code：无
- 完整报错/Complete Error Message：无

**Comment by ZHUxjj** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/114272296/3e0b8080-32ec-4f25-ac44-38fe873bad65)
我这样，一运行就退出，错误提示都没有。真坑

**Comment by beetter** [comment]
你好，这个问题你们找到解决方法了吗，我也是想把其中某个模型换成我自己训练的，不知道在哪里修改
